wasm-client/contracts/incentive/RewardsManager.sol

```diff
@@ -28,10 +28,10 @@ contract RewardsManager {
         return rewards[taskID];
     }
 
-    function depositReward(bytes32 taskID, uint reward, uint tax) public returns (bool) {
-        require(token.allowance(msg.sender, address(this)) >= reward + tax);
-        token.transferFrom(msg.sender, address(this), reward + tax);
-    
+    function depositReward(bytes32 taskID, uint reward, uint tax) internal returns (bool) {
+        // require(token.allowance(msg.sender, address(this)) >= reward + tax);
```

Comment:
We shouldnt comment these out, but I understand for testing purposes. 
This is related to #34

Reply:
I'm taking both the reward and tax from the deposit instead. So `npm run test test/os-wasm.js` should now work.

Reply:
Interesting.